sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -184,6 +186,29 @@ public void assign(final List<TopicPartition> assigned) {
               updateEndOffsets(ImmutableMap.of(tp, (long) records.get(tp).size()));
             }
           }
+          public Map offsetsForTimes(Map<TopicPartition, Long> timestampsToSearch) {
```

Comment:
* Can this return a non-raw type?
* Document that this is a non-`@Override` implementation of a particular method in Kafka 0.10